technic/doc/digilines.md

```diff
@@ -4,6 +4,10 @@
 This document describes the interaction of
 technic machines with the `digilines` mod (https://github.com/minetest-mods/digilines)
 
+## Digicables
+
+Digicables are a new type of cables that bring both electricity power and digiline-based information (analogic). Like both original types of cable (electric cable and digiline), you just have to connect cables to each other in order to transmit power/information. They are needed to access nuclear reactor digiline information (through the bottom). The digicables are available in HV, MV and LV.
```

Comment:
Maybe just drop "new", reactor does not have to be bottom, also few other suggestions.
```suggestion
Digicables are a type of cable that carry both electric power and digiline messages. Like both original types of cable (electric cable and digiline), you just have to connect cables to each other in order to transmit power/information. They are needed to access nuclear reactor digiline information. The digicables are available in HV, MV and LV.
```

I think important thing to add would be crafting because it needs alloying and therefore might not be that obvious especially if crafting guide isn't able to display alloying recipes (or if playing without guide).

Reply:
But then also, recipes maybe do not belong here in this file. Not sure if those are already somewhere but probably not...

Reply:
IMO it's better not to document recipes, because they can often be different. Servers change recipes quite often.

It should state the difference in connecting rules though, digicables connect differently than standard digilines.

Reply:
I've not actually ever seen a single change change in any of special recipes (that are not crafting which in turn works in all guides). Probably those too have changed outside the mods but definitely not often.

Reply:
Yeah, probably a bad assumption, but I have seen recipes changed. I do remember some technic recipes changed on Tunnelers' Abyss for example.

Reply:
> technic recipes changed on Tunnelers' Abyss for example.

They're not using either Technic plus nor original Technic mod, they're using their own fork which has their own documentation. Same goes with all of the servers I've seen changing their recipes.